pkg/processor/batchprocessor.go

```diff
@@ -90,22 +91,30 @@ func (s *segmentsBatch) poll() {
 			}
 			elapsed := time.Since(start)
 
+			batchesMap := make(map[string]string)
+			idRegexp := regexp.MustCompile(`\"id\":\"(.*?)\"`)
```

Comment:
Make this regexp a constant at the top level, we don't want to compile a regex every time

Reply:
ok